src/components/molecules/Settings/Account/PasswordModal/index.tsx

```diff
@@ -64,65 +130,71 @@ const PasswordModal: React.FC<Props> = ({ isVisible, onClose, hasPassword, updat
           large
           disabled={disabled}
           buttonType="primary"
-          text={intl.formatMessage({ defaultMessage: "Change your password now" })}
-          onClick={save}
+          text={intl.formatMessage({ defaultMessage: "Change password" })}
+          onClick={handleSave}
         />
       }>
       {hasPassword ? (
         <div>
-          <Text size="s" color={theme.main.text} otherProperties={{ margin: "22px auto" }}>
-            <p>
+          <Text size="m">
+            {intl.formatMessage({
+              defaultMessage: `In order to protect your account, make sure your password:`,
+            })}
+          </Text>
+          <SubText>
+            <Text size="m">
               {intl.formatMessage({
-                defaultMessage: `In order to protect your account, make sure your password:`,
+                defaultMessage: `* Is between 8 and 25 characters in length`,
               })}
-            </p>
-            <StyledList>
-              <li>
-                {intl.formatMessage({
-                  defaultMessage: `Is Longer than 8 characters`,
-                })}
-              </li>
-              <li>
-                {intl.formatMessage({
-                  defaultMessage: `At least 2 different numbers`,
-                })}
-              </li>
-              <li>
-                {intl.formatMessage({
-                  defaultMessage: `Use lowercase and uppercase letters`,
-                })}
-              </li>
-            </StyledList>
-          </Text>
-          <Label size="s">
-            {intl.formatMessage({ defaultMessage: "New password" })}
-            {/* {intl.formatMessage({
-                defaultMessage:
-                  "8 characters or more, 2 types or more from numbers, lowercase letters, uppercase letters",
-              })} */}
-          </Label>
-          <StyledTextBox
-            type="password"
-            borderColor={"#3f3d45"}
-            value={password}
-            onChange={setPassword}
-          />
-          <Label size="s">
-            {intl.formatMessage({ defaultMessage: "New password (for confirmation)" })}
-          </Label>
-          <StyledTextBox
-            type="password"
-            borderColor={"#3f3d45"}
-            value={passwordConfirmation}
-            onChange={setPasswordConfirmation}
-          />
+            </Text>
+
```

Comment:
Does this blank line mean anything to you?

Reply:
Nope!  I'll remove all unnecessary blanks!

Reply:
In HTML, blank lines are sometimes interpreted as whitespace; this doesn't happen in JSX, but it's confusing, isn't it?

Reply:
haha ya I don't like random spaces either unless there is good reason.